index.html

```diff
@@ -11,14 +11,6 @@
      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
 </head>
 <body>
-    <div class="phone-status">
-        <img src="/images/svg/Time Style.svg" alt="time">
-        <div class="left-status">
-            <img src="/images/svg/Cellular Connection.svg" alt="cellular-status">
-            <img src="/images/svg/Wifiwifi.svg" alt="wifi">
-            <img src="/images/svg/Battery.svg" alt="battery">
-        </div>
-    </div>
     <header id="navigation">
         <a href="#" class="logo">my logo</a>
```

Comment:
- [OPTIONAL] I would advise you to personalize the logo name.

Reply:
Sorry, I don't understand what you meant by personalizing the logo name.
`[OPTIONAL] I would advise you to personalize the logo name.`